assignment.md

```diff
@@ -34,30 +34,47 @@ Once you're finished with your edits, send the PR link to the HashiCorp recruite
 ---
 
 OVERALL QUESTIONS
-1. 
 2. 
 3. 
 
 ## Content
 -->
 
 <!-- EAJ TODO: Determine best structure for the document after working with the material. Ideas: Subheadings for each command; ol that contains an extended explanation; table that keeps command, description, and why to use it in rows -->
+# Managing code via the command line
+Git offers you many ways to manage your code both locally and remotely. Create branches to separate updates from production code, create patches, test ideas, and merge your changes into the codebase. The command line makes it easy to download, review, modify, and push changes between your local machine and a remote repository. <!-- New audiences may require a more in-depth intro, so we're leaving room for that here. -->
```

Comment:
Add an introduction that explains the purpose of the document. Provide a brief introduction (1-2 sentences) that gives a high-level overview of the information the document will provide. This way, users will understand at a glance whether this document is for them. Additional information is always welcome, but how much depends on your audience:

- Err on the side of providing more information and defining more for novices.
- Those who understand git basics may already know what local and remote branches and repositories are and their intended purposes.

I'm guessing that this document isn't intended for advanced users. You can always link to more information later that can lead these users to more advanced information that can define options and flags for a command.